docs/setup/new_environment.md

```diff
@@ -329,7 +329,7 @@ If you are using virtualbox, see the [Configuring VirtualBox for testing CommCar
 
 ### Testing your new CommCare Environment
 
-Run the following command to test each of the backing services as described [here](https://dimagi.github.io/commcare-cloud/commcare-cloud/deploy.html#step-3-checking-services-once-deploy-is-complete).
+Run the following command to test each of the backing services as described [here](../commcare-cloud/deploy.md#step-3-checking-services-once-deploy-is-complete).
```

Comment:
Is github.io going to get this right? Or will fixing this link for github.com break it for github.io?

Reply:
I'm pretty sure they both handle this well.  We've got a bunch of relative links like this.  I wasn't sure about the in-page anchor, which I copied from github.io, but I did confirm that it works also on github.com

Reply:
Confirming that this works as expected:
https://dimagi.github.io/commcare-cloud/setup/new_environment.html#testing-your-new-commcare-environment